sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -62,8 +62,14 @@
     c.client_net_address as client_address,
     sess.host_name as host_name,
     sess.program_name as program_name,
+    dbspu.user_objects_alloc_page_count as user_objects_alloc_page_count,
+    dbspu.user_objects_dealloc_page_count as user_objects_dealloc_page_count,
+    dbspu.internal_objects_alloc_page_count as internal_objects_alloc_page_count,
+    dbspu.internal_objects_dealloc_page_count as internal_objects_dealloc_page_count,
     {exec_request_columns}
 FROM sys.dm_exec_sessions sess
+    INNER JOIN sys.dm_db_session_space_usage dbspu
+           ON sess.session_id = dbspu.session_id
```

Comment:
Can you add the same change also to `IDLE_BLOCKING_SESSIONS_QUERY`? We split the activities for regular sessions and sleeping blockers into 2 queries